src/ontology/edits/oeo-shared-axioms.omn

```diff
@@ -3997,135 +3996,33 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1463"@en,
-        OEO_00040010 some OEO_00320001
-    
     
 Class: OEO_00320001
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A transport performance unit is a unit of measurement for the accumulated transport distance of a number of people and/or amount of goods.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289",
-        rdfs:label "transport performance unit"
-    
-    SubClassOf: 
-        <http://purl.obolibrary.org/obo/UO_0000000>
-    
     
 Class: OEO_00320002
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Passenger-kilometre is a transport performance unit for the accumulated transport distance of people where one passenger-kilometre equals the transport distance of 1 km for one person.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289
-
-Add pkm as synonym:
-issue: https://github.com/OpenEnergyPlatform/ontology/issues/1375
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1388",
-        <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "pkm",
-        rdfs:label "passenger-kilometre"
-    
-    SubClassOf: 
-        OEO_00320001
-    
     
 Class: OEO_00320003
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Ton-kilometre is a transport performance unit for the accumulated transport distance of goods where one ton-kilometre equals the transport distance of 1 km for one ton of goods.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289
-
-Add tkm as synonym:
-issue: https://github.com/OpenEnergyPlatform/ontology/issues/1375
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1388",
-        <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "tkm",
-        rdfs:label "ton-kilometre"
-    
     SubClassOf: 
         OEO_00320001
     
     
 Class: OEO_00320041
 
-    Annotations: 
-        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A vehicle operational mode is a realizable entity that determines how a vehicle is operating."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1304
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1314",
-        rdfs:comment "Examples are 'manual', 'partially automated', 'autonomous' or 'tele-operated'."@en,
-        rdfs:label "vehicle operational mode"@en
-    
     SubClassOf: 
         <http://purl.obolibrary.org/obo/BFO_0000020>,
         OEO_00010121 some OEO_00010023
```

Comment:
These SubClassOf axioms are duplicates and should be removed from oeo-shared-axioms